scissors/src/main/java/com/lyft/android/scissors/GlideFillViewportTransformation.java

```diff
@@ -17,16 +17,22 @@
 
 import android.graphics.Bitmap;
 import android.graphics.Rect;
+
 import com.bumptech.glide.load.engine.bitmap_recycle.BitmapPool;
 import com.bumptech.glide.load.resource.bitmap.BitmapTransformation;
 
+import java.nio.charset.Charset;
+import java.security.MessageDigest;
+
 class GlideFillViewportTransformation extends BitmapTransformation {
```

Comment:
Here are some more annoying changes. But the [`BitmapTransformation` documentation](https://github.com/bumptech/glide/blob/master/library/src/main/java/com/bumptech/glide/load/resource/bitmap/BitmapTransformation.java) is good and they include a use case there. That's where I got most of the code I wrote here.